### ISSUE TITLE
Skill Dispell is Bugs

### ISSUE THREAD

**Issue by cold-hr** [rathena/rathena#708]
I'm Use command
@ killable @ killer @ duel

and

Use skill dispell to player is not work

sorry I'm bad english

**Comment by officialronode** [comment]
Confirmed. Not working on @duel / @killer / @killable but it works fine on @pvpon  & @gvgon same as  on pvp maps and gvg maps

**Comment by cold-hr** [comment]
yes sir, I want to work for a normal map.

**Comment by KilleR** [comment]
Please remove the @killer tag from this thread.

If you are going to use @ for things, please enquote them, or use code
comments, thanks.

**Comment by cold-hr** [comment]
@KilleR  
Ok I remove youname  haha

**Comment by lighta** [comment]
what was the test context ?
What did you tryed @aleos89.
From what @officialronode say I would have rephrase this as SA_DISPELL only work in certain mapflag.
The "work" or "not work" definition would be nice to be sure but the only thing would might be mapflag related beside the obvious skill_nocast_db.txt, would be this in src. 
<code>
			if( sd && dstsd && !map_flag_vs(sd->bl.m) && sd->status.guild_id == dstsd->status.guild_id ) {
				clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0);
				break;
			}
</code>
Which would have display a failure message and would kick off only for same guildie (or both without guild). But can't say without context.

**Comment by cydh** [comment]
please use `code` markdown for any `@command`

**Comment by anacondaqq** [comment]
confirm.

**Comment by anacondaqq** [comment]
i was fix it manually by replacing this code

```
if( sd && dstsd && !map_flag_vs(sd->bl.m) && sd->status.guild_id == dstsd->status.guild_id ) { clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0); break; } 
```

to this one (i was add duels checks for src / dst target)

```
			if( sd && dstsd && (!map_flag_vs(sd->bl.m) || (sd && sd->duel_group && sd->duel_group == dstsd->duel_group)) && sd->status.guild_id == dstsd->status.guild_id ) {
				clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0);
				break;
			}
```

Tested, working fine, but i'm newbie in sources, and not really know what i'm doing. So, be afraid about this modification. Thx
